modules/websocket/packet_buffer.h

```diff
@@ -5,6 +5,8 @@
 /*                             GODOT ENGINE                               */
 /*                        https://godotengine.org                         */
 /**************************************************************************/
+/* Copyright (c) 2024-present Redot Engine contributors                   */
+/*                                          (see REDOT_AUTHORS.md)        */
```

Comment:
Unnecesary

Reply:
The Redot stance as far as I can tell is to change every copyright header to include those two lines. I think that is why its in here. But I have not been involved.